__tests__/setup-dotnet.test.ts

```diff
@@ -16,6 +16,7 @@ describe('setup-dotnet tests', () => {
     process.env.RUNNER_TOOL_CACHE = toolDir;
     process.env.DOTNET_INSTALL_DIR = toolDir;
     process.env.RUNNER_TEMP = tempDir;
+    process.env['INPUT_INCLUDE-PRERELEASE'] = 'false';
```

Comment:
Do we need this variable?

Reply:
> Do we need this variable?

yes

Reply:
`core.getBooleanInput('include-prerelease')` reads this env.
On GitHub Actions, it is automatically defined by `action.yml` (at `include-prerelease:default`).